### DISCUSSION TITLE
After setting -Dotel.instrumentation.common.default-enabled to false Agent did not send trace anymore

### DISCUSSION THREAD

**Discussion by chawol** [open-telemetry/opentelemetry-java-instrumentation#8005] [Q&A] [answered]
full config is here
```
-javaagent:C:\Devoloper\Tree\OpentelemetryCollector\opentelemetry-javaagent.jar
-Dotel.traces.exporter=jaeger
-Dotel.metrics.exporter=none
-Dotel.exporter.jaeger.endpoint=http://localhost:14250
-Dotel.instrumentation.common.default-enabled=false
-Dotel.instrumentation.opentelemetry-api.enabled=true
-Dotel.instrumentation.opentelemetry-instrumentation-annotations.enabled=true
-Dotel.instrumentation.external-annotations.enabled=true
-Dotel.instrumentation.methods.enabled=true
-Dotel.instrumentation.spring-webmvc.enabled=true
-Dotel.javaagent.debug=true
```

acording to this doc https://opentelemetry.io/docs/instrumentation/java/automatic/agent-config/ The section Enable manual instrumentation only it's should only Export span with @Withspan annotation But it's seems like export nothing

when set it  to true is working fine  but after setting to false is not working anymore and there is no exception.
Any Help?

**Comment by mateuszrzeszutek** [comment]
Hey @chawol ,

Can you prepare a repro scenario for your issue? A small app that shows the bug you're struggling with

**Comment by chawol** [comment, marked as answer]
Sorry , I figure out the problem 
Due to I use the wrong Annotation from Package io.opentelemetry.extension.annotations.WithSpan
